### ISSUE TITLE
no-homogenous-tags does not work with example tags

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#166]
In the same way that

``` gherkin
Feature: Feature with homogenous tags

@tag1 @tag2
Scenario: This is a Scenario name
  Then this is a then step

@tag1 @tag3
Scenario Outline: This is another Scenario name
  Then this is a then step
```
is reporting an error because of `tag1` exists in all the scenarios, this gherkin code

``` gherkin
Feature: Feature with homogenous tags

Scenario: This is a Scenario name
  Then this is a then step

Scenario Outline: This is a Scenario name
  Then this is a then step with variable <foo>
@tag1 tag2
Examples:
  | foo |
  | bar |
@tag1 @tag3
Examples:
  | foo |
  | rab |
```

should report an error because `tag1` exists in all of examples. Then, `tag1` could be moved to Scenario Outline.
